includes/definitions/discovery/siteboss550.yaml

```diff
@@ -0,0 +1,125 @@
+                    states:
+                        - { value: 0, generic: 0, graph: 1, descr: de-energized }
+                        - { value: 1, generic: 2, graph: 1, descr: energized }
+                    skip_values:
+                        -
+                            oid: esIndexPC
+                            op: '!='
+                            value: 6
+                        -
+                            oid: esPointValueStr
+                            op: 'not_in_array'
+                            value: ['Active/Energized', 'Inactive/Energized', 'Active/De-energized', 'Inactive/De-energized']
+                        -
+                            oid: esPointName
+                            op: '='
+                            value: unnamed
+                -
+                    oid: esPointTable
+                    value: esPointValueStr
+                    num_oid: '.1.3.6.1.4.1.3052.10.1.1.1.1.6.{{ $index }}'
+                    descr: esPointName
+                    group: Power Outputs
+                    state_name: powerOutput
+                    states:
+                        - { value: 0, generic: 0, graph: 1, descr: de-energized }
+                        - { value: 1, generic: 2, graph: 1, descr: energized }
+                    skip_values:
+                        -
+                            oid: esIndexPC
+                            op: '!='
+                            value: 6
+                        -
+                            oid: esPointValueStr
+                            op: 'not_in_array'
+                            value: ['On/Energized', 'Off/De-energized']
+                        -
+                            oid: esPointName
+                            op: '='
+                            value: unnamed
+        humidity:
+            data:
+                -
+                    oid: esPointTable
+                    value: esPointValueInt
+                    num_oid: '.1.3.6.1.4.1.3052.10.1.1.1.1.6.{{ $index }}'
+                    descr: esPointName
+                    skip_values:
+                        -
+                            oid: esIndexPC
+                            op: '!='
+                            value: 3
+        count:
+            data:
+                -
+                    oid: esPointTable
+                    value: esPointValueStr
+                    num_oid: '.1.3.6.1.4.1.3052.10.1.1.1.1.7.{{ $index }}'
+                    descr: esPointName
+                    skip_values:
+                        -
+                            oid: esIndexPC
+                            op: '!='
+                            value: 5
+                        -
+                            oid: esPointName
+                            op: '='
+                            value: unnamed
+        percent:
+            data:
+                -
+                    oid: fsStatusTable 
+                    value: fsStatusVolumePercentLevel
```

Comment:
This should be storage, not a health sensor.

Reply:
This OID is to monitor fuel level, not disk storage. I could re-create the percent sensor as a fuel sensor, it just seems very specific to this type of device